sockeye/data_io.py

```diff
@@ -1261,24 +1286,29 @@ def create_sequence_readers(sources: List[str], targets: List[str],
                             vocab_sources: List[vocab.Vocab],
                             vocab_targets: List[vocab.Vocab]) -> Tuple[List[SequenceReader], List[SequenceReader]]:
     """
-    Create source readers with EOS and target readers with BOS.
+    Create source readers with EOS and target readers with BOS if vocabularies are passed.
```

Comment:
The comment should be updated to match the code.
```suggestion
    Create source readers with EOS and target readers with BOS.
```